assets/js/navigation.js

```diff
@@ -6,27 +6,29 @@
  * Finally adds a class required to reveal the search in the handheld footer bar.
  */
 ( function() {
-	// Add class to footer search when clicked
-	jQuery( window ).load( function() {
-		jQuery( '.storefront-handheld-footer-bar .search > a' ).click( function(e) {
-			jQuery( this ).parent().toggleClass( 'active' );
-			e.preventDefault();
+	// Add class to footer search when clicked.
+	requestAnimationFrame( function() {
```

Comment:
Similar to homepage.js, this file is injected at the end of the document, so DOM is already available to be queried. We wait a RAF, but that shouldn't be needed.